src/main/java/life/qbic/portal/portlet/ImageInfo.java

```diff
@@ -0,0 +1,54 @@
+package life.qbic.portal.portlet;
+
+/**
+ * <short description>
+ * <p>
+ * <detailed description>
+ *
+ */
+public class ImageInfo {
+  private final long imageId;
+  private final String name;
+  private final byte[] thumbnail;
+  private final String size;
+  private final String timePoints;
+  private final String channels;
+
+  private ImageInfo() {
+    throw new AssertionError("Default constructor of " + ImageInfo.class + " is disabled.");
+  }
```

Comment:
Why not remove it? 

```suggestion
```
http://groovy-lang.org/objectorientation.html#_constructors as far as I understood groovy only provides an empty constructor if non is given

Reply:
This is not groovy but Java. I guess you could also leave it out but this indicates that the constructor should not be implemented.

Reply:
Sorry that I mixed that up. It can be resolved, thanks for clarifying

Reply:
Java does handle that the same as groovy. Here the default constructor is only created in case no other constructor is defined. We could remove the private constructor but I do not see any harm in keeping it.